exonum/src/node/mod.rs

```diff
@@ -148,6 +148,8 @@ pub struct NodeApiConfig {
     pub state_update_timeout: usize,
     /// Enable api endpoints for the `blockchain_explorer` on public api address.
     pub enable_blockchain_explorer: bool,
+    /// Enable private unsafe api
+    pub enable_unsafe_api: bool,
```

Comment:
Can we make this param, invisible in config, when it seted to `false`?

Reply:
`#[serde(default)]` is the key